index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/Main/scss/index.css">
+    <title>Super UI </title>
+</head>
+<body>
+    
+
+    <nav class="navigation">
+      
+          <h1>SUPER UI</h1>
+        <ul class="navigation__list">
+          <li ><a class="navigation__list--item "href="#">Home</a></li>
+          <li ><a class="navigation__list--item" href="#">Docs</a></li>
+          <li><a class="navigation__list--item" href="#">Dark</a></li>
+        </ul>
+      </nav>
+  <div class="container">
+    <img src="/Images/undraw_building_blocks_re_rwls.svg" alt="">
```

Comment:
In images it is good to add alt atributes for better accessibility

Reply:
done